test/onnx2mlir/CustomFnTest.cpp

```diff
@@ -91,6 +89,59 @@ int main(int argc, char *argv[]) {
   node->set_op_type("SquareFn");
 
   check(model_proto);
+}
+
+void testUseOfOnnxModelTypes() {
+  RegisterFunSchema();
+
+  ModelProto model_proto;
+  model_proto.set_ir_version(7);
+  auto *opset_version = model_proto.add_opset_import();
+  opset_version->set_domain(ONNX_DOMAIN);
+  opset_version->set_version(ONNX_OPSET_VERSION);
+
+  auto *graph = model_proto.mutable_graph();
+
+  auto float_type = TensorProto_DataType::TensorProto_DataType_FLOAT;
+  auto int_type = TensorProto_DataType::TensorProto_DataType_INT32;
+
+  auto *x = graph->add_input();
+  x->set_name("x");
+  auto *x_type = x->mutable_type()->mutable_tensor_type();
+  x_type->set_elem_type(float_type);
+  auto *x_shape = x_type->mutable_shape();
+  x_shape->add_dim()->set_dim_value(10);
+
+  auto *t = graph->add_value_info();
+  t->set_name("t");
+  auto *t_type = t->mutable_type()->mutable_tensor_type();
+  t_type->set_elem_type(int_type);
+  auto *t_shape = t_type->mutable_shape();
+  t_shape->add_dim()->set_dim_value(10);
+
+  auto *y = graph->add_output();
+  y->set_name("y");
+  auto *y_type = y->mutable_type()->mutable_tensor_type();
+  y_type->set_elem_type(float_type);
+  auto *y_shape = y_type->mutable_shape();
+  y_shape->add_dim()->set_dim_value(10);
+
+  auto *node = graph->add_node();
+  node->add_input("x");
+  node->add_output("t");
+  node->set_op_type("CustomFn1");
+
+  node = graph->add_node();
+  node->add_input("t");
+  node->add_output("y");
+  node->set_op_type("CustomFn2");
+
+  check(model_proto);
+}
+
+int main(int argc, char *argv[]) {
+  // testCustomFunTranslation();
```

Comment:
I'm guessing this test should be enabled?

Reply:
Yes, will do that. I think I did that because of the use of a static-variable that caused problems with doing two different translations, one after another (fixed in a different PR https://github.com/onnx/onnx-mlir/pull/418 ).

Reply:
Thanks. This looks ready to be merged in (I guess the linux-s390x CI is not required, is it?)